docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -3179,14 +3179,17 @@ are explained in more depth in the table below.
   :header-rows: 0
 
   * - Noise Function Affected
-    - Additional Inputs 
+    - Additional Input
     - Default Input Value 
   * - Typographic Noise
     - Probability that a corrupted token contains the original token (e.g., if typically e -> r, the probability that e -> er)
     - 0.1 
   * - Age miswriting 
     - Possible perturbations of age (e.g., for [1, -1] and age of 7, the possible "incorrect" results will be 6 and 8)
     - [1, -1] 
+  * - Age miswriting
```

Comment:
Not sure what's easier/clearer from the end user's perspective.

Reply:
I based the parameters for my version on the parameters for [numpy.random.Generator.choice](https://numpy.org/doc/stable/reference/random/generated/numpy.random.Generator.choice.html). That is, you pass a list-like of values (`a` in`Generator.choice`) and optionally pass a corresponding list-like of probabilities (`p` in `Generator.choice`). I like having `p` default to uniform probabilities when `p=None`, so that I have less to think about if I just want to try something out quickly.

If we do it this way, then yes, we need to clarify that when specifying probabilities, they must be in the same order as the increments.

If we want to have a more flexible user interface, we could optionally allow the user to pass a list of tuples or a dictionary for the age increments parameter, instead of having to specify the probabilities separately. But I think it's better to implement the simpler interface like in `Generator.choice` first, and we can add this feature later if we want.

Reply:
Since we are going to allow null/unspecified probabilities (for uniform choice), I like having them as separate inputs. I have added a note that the order is critical.